static_src/test/unit/actions/service_actions.spec.js

```diff
@@ -6,6 +6,15 @@ import cfApi from '../../../util/cf_api.js';
 import serviceActions from '../../../actions/service_actions.js';
 import { serviceActionTypes } from '../../../constants.js';
 
+function assertAction(spy, type, params) {
+  expect(spy).toHaveBeenCalledOnce();
+  let arg = spy.getCall(0).args[0];
```

Comment:
with that said. i'm fine with it for now, and we can worry about it later when that time comes.